pkg/kor/replicaset.go

```diff
@@ -13,6 +13,19 @@ import (
 	"github.com/yonahd/kor/pkg/filters"
 )
 
+// These replica sets generated for default GKE
+// TODO: Need to add it the ability for isResourceException to handle regex/wildcard matching
```

Comment:
GKE creates empty replicasets?

Reply:
Yeah, I just re-created a new cluster again and these are not there. Going to remove them.

Reply:
Depending on if you build a `standard` cluster or use `autopilot` replicasets are created. For now we can just leave this as is.

Reply:
GKE includes Prometheus for metrics out of the box, so this is probably why these exists.

Reply:
Let's remove this for now.
I'm not sure why they would be install empty replicasets and in any case I don't see harm in cleaning it. The deployment will recreate one when it is scaled up